### PR TITLE
docs: update terms in dictionary

### PR DESCRIPTION
**Pull request by lirantal**

### CODE REVIEW

File: ci/spelling-config.json [hunk 1]
```diff
@@ -113,6 +113,15 @@
         "backdoors",
         "EIOPA",
         "ESMA",
-        "SSDF"
+        "SSDF",
+        "CVE",
+        "PEAR",
+        "PHP",
+        "TAR",
+        "unpacthed",
```
Comment (mnm678) [inline ci/spelling-config.json:121]: should this be "unpatched"?

Comment (SantiagoTorres) [reply ci/spelling-config.json:121]: ping @lirantal fix this pls :)

Comment (PushkarJ) [inline ci/spelling-config.json:121]:
```suggestion
        "unpatched",
```

**Comment by SantiagoTorres** [comment]
can I get a dco fix? :)

**Pull request merged**